parser/src/ast.rs

```diff
@@ -186,6 +186,12 @@ pub enum ExpressionType {
         values: Vec<Expression>,
     },
 
+    /// A named expression aka. assignment expression aka. "Walrus operator"
```

Comment:
The fact that the PEP and the implementation use different terminology is somewhat irritating.